### ISSUE TITLE
Echo Arena won't launch: "Failed to Initialize Oculus VR Session"

### ISSUE THREAD

**Issue by Zebwen** [LibreVR/Revive#703]
Launching Echo Arena. This issue occurs since 1.1.9. 1.1.8 works fine, 1.1.9, 1.1.10 and 1.1.11 all have this problem for me. Tried with both SteamVR standard and Beta. Tried injecting and patching from Revive dashboard. Tried running Steam as admin. Rebooting, restarting Oculus services, nothing helps unfortunately. Rolling back to 1.1.8 allows me to load the game again without issues. Both controllers and HMD running on launch attempt.

![image](https://user-images.githubusercontent.com/30393101/28502340-090bc206-6fe8-11e7-9d12-397a63876edc.png)

Happy to try any suggestions.

NOTE:
Not limited to Echo arena. Same happens with Farlands (and all other titles): 
![image](https://user-images.githubusercontent.com/30393101/28502374-c0824c7a-6fe8-11e7-9222-b21852b9114a.png)

```
Echo Arena log:
Launched injector with: "C:\Program Files\Revive\Revive\ReviveInjector_x64.exe"  D:/Games/Software/ready-at-dawn-echo-arena/bin/win7/EchoArena.exe
Creating process: D:/Games/Software/ready-at-dawn-echo-arena/bin/win7/EchoArena.exe 
Injecting DLL: C:\Program Files\Revive\Revive\x64\openvr_api.dll
LoadLibrary found at address: 0xdafee490
Injecting DLL: C:\Program Files\Revive\Revive\x64\LibRevive64_1.dll
LoadLibrary found at address: 0xdafee490
LoadLibrary failed to return module handle
```

```
Farlands Log:
Launched injector with: "C:\Program Files\Revive\Revive\ReviveInjector_x64.exe"  /library "Software\oculus-farlands\ww\Binaries\Win64\ww-Win64-Shipping.exe" -vr
Creating process: D:\Games\Software\oculus-farlands\ww\Binaries\Win64\ww-Win64-Shipping.exe -vr 
Injecting DLL: C:\Program Files\Revive\Revive\x64\openvr_api.dll
LoadLibrary found at address: 0xdafee490
Injecting DLL: C:\Program Files\Revive\Revive\x64\LibRevive64_1.dll
LoadLibrary found at address: 0xdafee490
LoadLibrary failed to return module handle
```

**Comment by Armada651** [comment]
> LoadLibrary failed to return module handle

This means Revive didn't load correctly, no idea why yet.

**Comment by Armada651** [comment]
Have you tried updating your graphics drivers? I think it's failing to load because a dependency on Vulkan was added. If you haven't update your drivers in a long while you may not have Vulkan on your system yet.

**Comment by Armada651** [comment]
Closed as duplicate of #702.
